### PR TITLE
[Composer] Handling versions

### PR DESCRIPTION
**Pull request by greedybro**

### CODE REVIEW

File: roles/composer/tests/0110_install_version.yml [hunk 1]
```diff
@@ -3,15 +3,70 @@
...
+          command: >
+            goss --gossfile {{ test }}.goss.yml --vars-inline "{tags: [major_version_no_change]}" validate
+
+    - tags: [major_version]
+      block:
+        - import_role:
+            name: manala.composer
+            tasks_from: install
+          vars:
+            manala_composer_version: '2'
```
Comment (nervo) [inline roles/composer/tests/0110_install_version.yml:68]: Why quoting 2 ?

Comment (greedybro) [reply roles/composer/tests/0110_install_version.yml:68]: to ensure it's working with both integers and strings

File: roles/composer/defaults/main.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ manala_composer_install_packages_default:
   - unzip
 
 # Version
-manala_composer_version: ~
+manala_composer_version: stable
```
Comment (nervo) [inline roles/composer/defaults/main.yml:11]: Please, reflect in README.md :)
And add a little bafouille on major channel/version/whatever handling

File: roles/composer/tests/0110_install_version.yml (hunk 1 above)
Comment (nervo) [reply roles/composer/tests/0110_install_version.yml:68]: Don't you cast it as a string when using it ?

Comment (greedybro) [reply roles/composer/tests/0110_install_version.yml:68]: yup, can remove it but since it's in the test ¯\_(ツ)_/¯
your call

Comment (nervo) [reply roles/composer/tests/0110_install_version.yml:68]: Ok, thats fine 👍

File: roles/composer/defaults/main.yml (hunk 2 above)
Comment (greedybro) [reply roles/composer/defaults/main.yml:11]: done

**Pull request merged**